docs/drafts/index.html

```diff
@@ -75,6 +205,12 @@ <h2>Introduction</h2>
 				<li>References to the corresponding normative statements or sections in the EPUB 3.3 specification(s).</li>
 				<li>Pointer to the (consolidated) implementation status of the test in the separate <a href="results">implementation report</a>.</li>
 			</ol>
+
+			<p>
+				Strictly speaking, the “<code>should</code>” and “<code>may</code>” tests are not necessary for the official CR testing of
```

Comment:
I don't think you need both quotation marks and <code>; the <code> acts as implicit punctuation.

Reply:
+1